mdata/cache/accnt/if.go

```diff
@@ -20,3 +20,8 @@ type EvictTarget struct {
 	Metric string
 	Ts     uint32
 }
+
+func NewAccnt(maxSize uint64) Accnt {
+	// currently we only know NewFlatAccnt
+	return NewFlatAccnt(maxSize)
+}
```

Comment:
constructors should generally not return an interface, but rather the concrete type.  and you could call the constructor from the tests and access the internal variables.